rmf_fleet_adapter_python/src/planner/planner.cpp

```diff
@@ -144,4 +153,44 @@ void bind_plan(py::module &m) {
                     py::overload_cast<>(&Plan::Goal::orientation, py::const_),
                     py::overload_cast<double>(&Plan::Goal::orientation))
       .def("any_orientation", &Plan::Goal::any_orientation);
+
+  // Configuration =============================================================
+  py::class_<Configuration>(m_plan, "Configuration")
+      .def(py::init<Graph, VehicleTraits>(),
+           py::arg("graph"),
+           py::arg("traits"))
+      .def_property_readonly("graph", &Configuration::graph)
+      .def_property_readonly("traits", &Configuration::vehicle_traits);
+  
+  // Options =============================================================
+  // TODO
+  // Light wrappers
+  py::class_<Interpolate::Options>(m, "InterpolateOptions")
+      .def(py::init<>());
```

Comment:
Looks fine to me. One minor point, should we just remove binding for `Interpolate::Options` for now since it is not being used in the `Configuration` yet

Reply:
Done https://github.com/open-rmf/rmf_ros2/pull/11/commits/e9ecb827bd33f0668d80cbda7d02ca5c56f4f76f